index.html

```diff
@@ -19,30 +19,37 @@
         </div>
     </header>
     <main>
-        <section class="main-section">
+        <section class="main-section" id = "pokemon-section">
             <img class="main-logo" src="https://images.unsplash.com/photo-1605979257913-1704eb7b6246?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=2550&q=80" />
             <h2>
                 Welcome to Pokemon Catcher!
             </h2>
-            <div>
+           <div class = "pokemon-div" id = "pokemon-div">
+            
+            <div class = "card-div">
             <label> 
```

Comment:
comment out all the html in side this pokemon-div, lines 30-39 because you are rendering those with your renderNewPokemon function